compiler/il/OMRBlock.hpp

```diff
@@ -162,7 +162,8 @@ class OMR_EXTENSIBLE Block : public TR::CFGNode
 
    TR::Block * split(TR::TreeTop * startOfNewBlock,  TR::CFG * cfg, bool fixupCommoning = false, bool copyExceptionSuccessors = true, TR::ResolvedMethodSymbol *methodSymbol = NULL);
    
-   TR::Block * splitPostGRA(TR::TreeTop *startOfNewBlock, TR::CFG *cfg, bool copyExceptionSuccessors = true, TR::ResolvedMethodSymbol *methodSymbol = NULL);
+   TR::Block * splitPostGRA(TR::TreeTop *startOfNewBlock, TR::CFG *cfg, bool copyExceptionSuccessors = true, TR::ResolvedMethodSymbol *methodSymbol = NULL,
+                            bool fixupCommoning = true, bool isCopyLiveLocals = true);
```

Comment:
I believe using this to split the block with `fixupCommoning` set to `false` would be a common use, Can we change default value to false?

Reply:
Setting `fixupCommoning` as `true` by default is the today's default usage of `splitPostGRA`, such as in the following OpenJ9 code locations [1]. Another reason of choosing the default value as `true` in OMR is to separate the dependency on where it's used in OpenJ9.  If it's set as `false` by default, it'll break the following existing OpenJ9 code [1] since they expect uncommoning and handling the GlRegDeps.  

The corresponding OpenJ9 change is being addressed in https://github.com/eclipse/openj9/pull/10976.  

[1]
https://github.com/eclipse/openj9/blob/2ce44b344232bebb32ce4dfe22390c0a77ccf737/runtime/compiler/codegen/J9CodeGenerator.cpp#L253
https://github.com/eclipse/openj9/blob/2ce44b344232bebb32ce4dfe22390c0a77ccf737/runtime/compiler/codegen/J9CodeGenerator.cpp#L1628

Reply:
Yeah I was thinking about that only. That behaviour is expected in JProfilingValue as well. I think it should be fine with the scope of this PR as it guarantees no build breakage.